api/v1alpha1/secretagentconfiguration_types.go

```diff
@@ -191,6 +191,9 @@ type AppConfig struct {
 	AWSRegion             string         `json:"awsRegion,omitempty"`
 	AzureVaultName        string         `json:"azureVaultName,omitempty"`
 
+	// Optional timeout value to generate a individual secret. Defaults to 40
+	SecretTimeout *int `json:"secretTimeout,omitempty"`
```

Comment:
Can you also add this to:
* sample config in _this_ repo
* secret_agent_config.yaml - in forgeops

Reply:
yes, will do

Reply:
Its in the sample config but commented out
`    ## Timeout in seconds for generating each individual secret. Default 40
    # secretTimeout: 40 `

Reply:
sorry about that, is it in forgeops too?

Reply:
I've created a separate ticket for that, I'll do that today